frontend/src/queries/nodes/DataTable/DataTableExport.tsx

```diff
@@ -39,12 +43,141 @@ function startDownload(query: DataTableNode, onlySelectedColumns: boolean): void
     })
 }
 
+const columnDisallowList = ['person.$delete', '*']
+const getCsvTableData = (dataTableRows: DataTableRow[], columns: string[], query: DataTableNode): string[][] => {
+    if (isPersonsNode(query.source)) {
+        const filteredColumns = columns.filter((n) => !columnDisallowList.includes(n))
+
+        const csvData = dataTableRows.map((n) => {
+            const record = n.result as Record<string, any> | undefined
+            const recordWithPerson = { ...(record ?? {}), person: record?.name }
+
+            return filteredColumns.map((n) => recordWithPerson[n])
+        })
+
+        return [filteredColumns, ...csvData]
+    }
+
+    if (isEventsQuery(query.source)) {
+        const filteredColumns = columns
+            .filter((n) => !columnDisallowList.includes(n))
+            .map((n) => extractExpressionComment(n))
+
+        const csvData = dataTableRows.map((n) => {
+            return columns
+                .map((col, colIndex) => {
+                    if (columnDisallowList.includes(col)) {
+                        return null
+                    }
+
+                    if (col === 'person') {
+                        return n.result?.[colIndex]?.properties?.email || n.result?.[colIndex]?.uuid
```

Comment:
Wanted to double-check whether the `uuid` was the correct alternative for anonymous users?